lms/static/js/learner_dashboard/views/subscription_upsell_view.js

```diff
@@ -0,0 +1,30 @@
+import Backbone from 'backbone';
+
+import HtmlUtils from 'edx-ui-toolkit/js/utils/html-utils';
+
+import subscriptionUpsellTpl from '../../../templates/learner_dashboard/subscription_upsell_view.underscore';
+
+class SubscriptionUpsellView extends Backbone.View {
+    constructor(options) {
+        const defaults = {
+            el: '.js-subscription-upsell',
+        };
+        super(Object.assign({}, defaults, options));
+    }
+
+    initialize(options) {
+        this.tpl = HtmlUtils.template(subscriptionUpsellTpl);
+        this.data = options.context;
+        this.render();
+    }
+
+    render() {
+        const data = $.extend(this.context, {
+            minSubscriptionPrice: '$39',
+            trialLength: 7,
```

Comment:
Similarly to the last ticket you had us review, are these values expected to be placed behind a Django setting? 
https://github.com/openedx/edx-platform/pull/32060#discussion_r1183121493

Reply:
yes, that's the plan